### PR TITLE
fix(sip-connector): set correct context for generate audio stream method

### PR DESCRIPTION
**Pull request by papamishka911**
Set correct context for generate audio stream method + fix for session.mock (create peerconnection with audio stream when incoming mediaStream has no video tracks

### CODE REVIEW

File: src/__mocks__/Session.mock.ts [hunk 1]
```diff
@@ -52,14 +52,26 @@ class Session extends BaseSession {
     return true;
   }
 
+  hasVideoTracks(mediaStream: MediaStream): boolean {
```
Comment (Krivega) [inline src/__mocks__/Session.mock.ts:55]: don't create method, use function (this is not used)

File: src/__tests__/callWithNoVideoTracks.ts [hunk 2]
```diff
@@ -46,13 +45,25 @@ describe('call with no video tracks', () => {
...
+
+  it('should no exist video tracks in incoming mediaStream when call with no video', async () => {
+    expect.assertions(1);
+
+    await sipConnector.connect(dataForConnectionWithAuthorization);
+    await sipConnector.call({ number, mediaStream, ontrack: mockFn });
+
+    const remoteStreams = sipConnector.getRemoteStreams();
+
+    // @ts-ignore
```
Comment (Krivega) [inline src/__tests__/callWithNoVideoTracks.ts:64]: remove

Comment (papamishka911) [reply src/__tests__/callWithNoVideoTracks.ts]: done

File: src/__mocks__/Session.mock.ts (hunk 1 above)
Comment (papamishka911) [reply src/__mocks__/Session.mock.ts]: done

**Pull request merged**